rbc/tests/test_omnisci_column_list.py

```diff
@@ -71,3 +71,24 @@ def test_columns_sum(omnisci, T, variant):
     actual = list(zip(*result))[0]
 
     assert expected == actual
+
+
+@pytest.mark.xfail
+def test_columnlist_enumerate(omnisci):
+
+    # @omnisci('int32(Cursor<ColumnList<T>>,'
+    #          ' RowMultiplier, OutputColumn<T>)',
+    #          T=['int32'], devices=['cpu'])
+    # def columnlist_enumerate(lst, m, out):
+    #     for j in range(lst.ncols):
+    #         col = lst[j]
+    #         for i, e in enumerate(col):
+    #             out[i] = e if j == 0 else out[i] + e
+    #         # for i in range(lst.nrows):
+    #         #     out[i] = col[i] if j == 0 else out[i] + col[i]
+    #     return lst.nrows
+
+    query = (f'select * from table(columnlist_enumerate(cursor('
+             f'select i4, i4, i4 from {omnisci.table_name}), 1))')
+    _, result = omnisci.sql_execute(query)
+    assert list(result) == [(0,), (3,), (6,), (9,), (12,)]
```

Comment:
Could you store this test function in https://github.com/xnd-project/rbc/issues/442 and remove it here to keep the tests clean of dead code?

Reply:
done